modules/web-crypto-backend/test/synchronous_random_values.test.ts

```diff
@@ -18,11 +18,28 @@
 import { expect } from 'chai'
 import 'mocha'
 import { synchronousRandomValues } from '../src/index'
+import sinon from 'sinon'
+import * as browserWindow from '@aws-sdk/util-locate-window'
```

Comment:
Be careful about doing a * import statement, could involve importing unneeded things

Reply:
:) . However, I need to stub the module.  That is why I'm doing it

Reply:
Oh okay, so it's specific for the sinon import (per a Google search), that makes more sense